javascript/javascript3/week3/app.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <title>Document</title>
```

Comment:
```suggestion
    <title>Screenshot App</title>
```
Something that would be as accurate and meaningful as possible!
Good reading https://www.w3schools.com/tags/tag_title.asp